projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -144,6 +166,25 @@ export class TableHeaderCellRendererComponent implements OnInit, OnChanges {
     this.sortChange.emit(direction ?? this.getNextSortDirection(this.sort));
   }
 
+  public getTooltip(
+    titleTooltip: string | undefined,
+    title: string | undefined
+  ): string | TemplateRef<unknown> | undefined {
+    if (titleTooltip === undefined) {
+      return title;
+    }
+
+    return this.htmlTooltipTemplate;
+  }
+
+  public getSanitizedHtml(tooltipString: string | undefined): string | null {
+    if (tooltipString === undefined) {
+      return '';
+    }
+
+    return this.sanitizer.sanitize(SecurityContext.HTML, tooltipString);
```

Comment:
if we keep the sanitizer, we should put the `?? undefined` here - our methods should never return null (I thought we had a lint rule on that)